README.md

```diff
@@ -139,6 +141,51 @@ const moduleConfig: OktaConfig = { oktaAuth };
 export class MyAppModule { }
 ```
 
+### `APP_INITIALIZER`
+
+Starting with `okta-angular 6.2.0`, you can provide `OktaConfig` in `APP_INITIALIZER` provider factory with method `setConfig()` of `OktaAuthConfigService` instance.
+
+```typescript
+// myApp.module.ts
+
+import {
+  OktaAuthModule,
+  OktaConfig,
+  OktaAuthOptions,
+  OktaAuthConfigService,
+} from '@okta/okta-angular';
+import { OktaAuth } from '@okta/okta-auth-js';
+
+function configInitializer(configService: OktaAuthConfigService, httpBackend: HttpBackend) {
```

Comment:
Since this is in the documentation, I recommend changing this to the following:

```ts
function configInitializer(configService: OktaAuthConfigService, httpBackend: HttpBackend): () => void {
  return () =>
  new HttpClient(httpBackend)
    .get('api/config')
    .pipe(
      map((res: any) => ({issuer: res.issuer, clientId: res.clientId, redirectUri: window.location.origin + '/login/callback'}))
      tap((authConfig: OktaAuthOptions) => {
        const oktaAuth = new OktaAuth(authConfig);
        const moduleConfig: OktaConfig = { oktaAuth };
        configService.setConfig(moduleConfig);
      }),
      take(1)
  );
}
```

That way, it will make it clear about the mapping between HttpResponse to the minimum expected config type